probe-rs/tests/inlined_function.rs

```diff
@@ -1,12 +1,15 @@
-use probe_rs::debug::{debug_info::DebugInfo, DebugError};
+use probe_rs::{
+    debug::{debug_info::DebugInfo, DebugError},
+    RegisterValue,
+};
 
 type TestResult = Result<(), DebugError>;
 
 #[test]
 fn function_name_of_inlined_function_1() -> TestResult {
     let di = DebugInfo::from_file("tests/inlined-function").unwrap();
 
-    let address = 0x15e;
+    let address = RegisterValue::U32(0x15e);
```

Comment:
I don't love this pattern as it requires the caller to pack the type.  If we keep `RegisterValue` as the input to the functions maybe it would be better to take `impl Into<RegisterValue>` so the caller can still pass a u32 or u64 seamlessly.

Reply:
Point taken ... I will revert to using address.